cpp/src/slice2java/Gen.cpp

```diff
@@ -2928,7 +2928,7 @@ Slice::Gen::TypesVisitor::visitExceptionEnd(const ExceptionPtr& p)
         out << sp;
         writeDocComment(
             out,
-            "@{inheritDoc}\n"
```

Comment:
This '@' sign has always been in the wrong place, and we just never noticed apparently.